make/ncurses.mk

```diff
@@ -105,6 +106,7 @@ $(NCURSES_DIR)/.configured: $(DL_DIR)/$(NCURSES_SOURCE) $(NCURSES_PATCHES) make/
 		--host=$(GNU_TARGET_NAME) \
 		--build=$(GNU_HOST_NAME) \
 		--prefix=$(TARGET_PREFIX)	\
+		--with-termlib		\
```

Comment:
Generating a separate terminfo library will break many Optware packages, so we don't want this.